Mobile/ShopLens/Droid/Activities/CameraActivity.cs

```diff
@@ -218,11 +222,23 @@ private void RecogniseImage(Uri uri)
                         QueueMode.Flush,
                         null,
                         null);
-
+                    new ErrorDialogCreator(this, "Shopping cart", "Would you like to add to thid product to your shopping cart?", "Yes", "No", 
+                        addToShoppingCart, doNotAddToShoppingCart);
+                    new MessageBarCreator(rootView, "Product was added.");
                 }
             });
         }
 
+        public void addToShoppingCart()
+        {
+
+        }
+
+        public void doNotAddToShoppingCart()
```

Comment:
Add item to Cart ? => Yes => Call method to add item - OK, but it should use MLau's way of adding items to cart then, it should call his method.
Add item to Cart ? => No => Call method to not add item to Cart - Seems weird, unless there's a good reason to call the do not add method, but I can't think of one right now.